app/Services/Caches/CampaignCacheService.php

```diff
@@ -244,10 +244,13 @@ public function themes(): string|bool
             ->get();
         /** @var CampaignPlugin $plugin */
         foreach ($plugins as $plugin) {
-            $theme .= "/** plugin: " . e($plugin->name) . " #" . e($plugin->version->version) . " **/\n";
-            $theme .= $plugin->version->fonts . "\n\n";
+            $theme .= "/** plugin: " . e($plugin->name) . " #" . e($plugin->version->version) . " fonts **/\n";
```

Comment:
Sorry for not making it clear, only add the code and comment **if** there is a font. Otherwise my local setup starts with 16 comments but no content, which isn't useful :)